docs/source/dummies_node.rst

```diff
@@ -0,0 +1,650 @@
+
+`id` is an Integer, it's also the `primary_key` of the table which means that no two rows can have the same value. `Creation_time` is a time, it can't be null (i.e. all filled rows must have a value) and unless you tell it otherwise it will be filled with whatever the time was when the row was filled (that's the `default=timenow` bit).
+
+After this are a bunch of property columns:
+::
+
+	#: a generic column that can be used to store experiment-specific details in
+    #: String form.
+    property1 = Column(Text, nullable=True, default=None)
+
+These can be used for anything you feel like, we'll see some examples of this later on.
+
+Next come `failed` and `time_of_death`:
+::
+
+    #: boolean indicating whether the Network has failed which
+    #: prompts Dallinger to ignore it unless specified otherwise. Objects are
+    #: usually failed to indicate something has gone wrong.
+    failed = Column(Boolean, nullable=False, default=False, index=True)
+
+    #: the time at which failing occurred
+    time_of_death = Column(DateTime, default=None)
+
+`failed` is used to mark rows as, well, failed and `time_of_death` simply records the time at which this failing occurred. Rows start off unfailed (i.e. their `failed` value is False), but once rows are marked as failed (i.e. their `failed` value is set to True) Dallinger will ignore them from then on, unless told otherwise. For instance, if you ask how many nodes are in a network, Dallinger will tell you how many *unfailed* nodes are in the network. Similarly, if you ask for all the nodes associated with a particular participant, Dallinger will give you a list of all the *unfailed* nodes of that participant.
+
+Why would you want to fail a node? Well let's say you a participant spills coffee on their computer half way through the experiment and they disappear. You recruit another participant to take their place, but you now need a way to get rid of the incomplete data from the earlier participant. This is what failing is for - the data isn't deleted, but, unless you tell it otherwise, Dallinger will continue with the experiment as if those rows in the table were not there. There's a thousand reasons you might want to fail a participant and we'll see many more of them later on in this guide.
+
+The final column is `details`. This serves a very similar function to the property columns discussed above, but is fancier and generally better. Chances are that down the line `details` will entirely replace the property columns and so this bit of the guide will need to be rewritten.
+
+Node objects
+------------
+
+So far we've covered Node's from the table view, but remember that all Dallinger classes have table/object duality and in general the object side of things is far more useful. So what are the extra features of Nodes if we treat them as objects? (In a good way.) Let's return to the Node class in models.py and look immediately below where the columns were created. The first function is `__init__`:
```

Comment:
"(In a good way.)"
I am not following this part of the comment? :thinking:

Reply:
Is this a reference to treating Nodes as if not having a soul or something? :laughing:

Reply:
I'm not sure either - this is what happens when I binge write documentation! Me and Jesse have been tweaking the docs on the branch `jesse-edits` feel free to remove it there (Jesse, who did the dummies_node tweaks may have already done so).

Reply:
yeah, I'm working my way through that branch and making tweaks. Much ground to cover :muscle: